src/Mesh/Meshing.jl

```diff
@@ -1,6 +1,6 @@
 
 """
-    generate_mesh(proj::Project)
+    generate_mesh(proj::Project; verbose::Bool, subdivision_maximum::Integer)
```

Comment:
```suggestion
    generate_mesh(proj::Project; verbose=false, subdivision_maximum=8)
```
I think this should not be more restrictive regarding types than the actual function. Instead, I'd rather document the default values